react/components/FilterBar/index.js

```diff
@@ -217,6 +219,7 @@ export const filterBarPropTypes = {
   submitFilterLabel: PropTypes.string,
   /** New Filter title label for inside the 'More options' menu */
   newFilterLabel: PropTypes.string,
+  testIdMoreOptions: PropTypes.string,
```

Comment:
Why do we need to pass it by props every time? 🤔. I think the internal components should have static ids.

Reply:
Because we can use the component more than once in a app, then, they should have a unique key, the id.

Reply:
Got it, makes sense.